### PR TITLE
[scopes][sites]Add admin scopes to CUD routes

### PR DESCRIPTION
**Pull request by fe51**
In the issue #45 we decide to define scopes for all routes in order to protect the API

This PR aims at adding scopes for sites route.

However, should we add the right to any auth user to CU sites ? What do you think ?
(I would add "me" in scopes in this case)

Any feedback is welcome !

### CODE REVIEW

**Comment by fe51** [comment]
** tests need to be updated**

**Review by frgfm — APPROVED** [review]
Looks good to me! Like in the other PR, let's wait for another review. And about your question, modifying/creating sites should be validated by a system administrator, I think it would be dangerous allowing an authenticated user to get those rights.

**Comment by frgfm** [comment]
@jeanpasquier75 @florianriche @martin1tab We'd need your inputs on this PR please :pray:

**Review by jeanpasquier75 — APPROVED** [review]

**Pull request merged**
